src/server/utils/logging/config/development.js

```diff
@@ -29,11 +29,12 @@ export const pinoPrettyOptions = {
     debug: 20,
     info: 30,
     log: 35,
+    dev: 36,
     warn: 40,
     error: 50,
     fatal: 60,
   },
-  customColors: 'trace:white,debug:green,info:gray,log:blue,warn:yellow,error:red,fatal:bgRed',
+  customColors: 'trace:white,debug:green,info:gray,log:blue,dev:green,warn:yellow,error:red,fatal:bgRed',
```

Comment:
Is there a color we can use that we're not already using?

Maybe we _should_ use magenta.

Reply:
i like magenta. but where do I find the list of accepted colors? I was trying with "orange" or "purple" but those didnt work